metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/adapters/geo/GeoAdapter.java

```diff
@@ -46,7 +46,6 @@ public JSONObject enrich(CacheKey value) {
     JSONObject enriched = new JSONObject();
     Optional<HashMap<String, String>> result = GeoLiteDatabase.INSTANCE.get(value.coerceValue(String.class));
     if(!result.isPresent()) {
-      _LOG.error("GEO Enrichment failure: {}", value.coerceValue(String.class));
```

Comment:
Can you make it log.debug()?  I think it's still useful to be able to tell what happened, it just shouldn't be error.